### DISCUSSION TITLE
Certain slider lengths aren't equal to stable lengths

### DISCUSSION THREAD

**Discussion by apollo-dw** [ppy/osu#15106] [Q&A] [answered]
To reproduce:
1. Download [this linked map](https://osu.ppy.sh/beatmapsets/1584926#osu/3237399) on both lazer and stable.
2. Open the top difficulty
3. Refer to 00:25:305 (1,1) - 

Screenshots provided show the difference, top is stable and bottom is lazer.
![image](https://user-images.githubusercontent.com/83023433/137309972-0860ed5b-cd6d-4b27-9528-87b7533cfd38.png)
![image](https://user-images.githubusercontent.com/83023433/137310033-c95fb922-93db-49b0-92ba-555b347fa3fe.png)

Tested with ppy/master as well to see if the new difficulty points stuff changes behaviour - doesn't seem to be the case.

Very important to note that this somehow affects difficulty calculation, as far as I can tell - which is important since this bug can affect slider-related calculations.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15109
